lib/library.go

```diff
@@ -106,6 +106,15 @@ func CallRPC(inputJSON *C.char) *C.char {
 	return C.CString(outputJSON)
 }
 
+//SeedWhisperSymKey seeds a random mnemonic passphrase and a Whisper symmetric
+//key seeded with that passphrase as its password, returned via output function
+//parameters.
+//export SeedWhisperSymKey
+func SeedWhisperSymKey(passphrase *C.char, symKeyID *C.char) {
```

Comment:
i don't like the idea of extending library interface with such helpers, if they are needed i would suggest to add another json rpc api (statusdebug). do we need this API for production or only for tests? 

cc @adambabik

Reply:
This was asked by @yenda and this is for production usage.

Account manager's methods `SelectAccount, ReSelectAccount, Logout` changes the state of a whisper. And all of them are exposed in the library. I think this PR is about exposing one more method from Account manager.

Reply:
oh well, you are right